vm/bin/user-bootstrap.sh

```diff
@@ -10,6 +10,3 @@ bash /vagrant/bin/misc-install.sh
 
 # Install P4lang tools
 bash /vagrant/bin/install-p4-tools.sh
-
-# make sure .cache is not own by root
-sudo chown -R p4:p4 /home/p4/.cache/
```

Comment:
This command was failing when I tried to run `vagrant up`, after commenting out the non-default TCP port usage for ssh in the first diff above.